### PR TITLE
Fix math rendering in docstrings

### PR DESCRIPTION
**Pull request by natashawatkins**
Sorry about such a large commit (did not expect to fix this much)

A lot of math renderings weren't working either due to:
- no use of .. math::
- not using raw string
- not using '\\' for symbols
- matrices formatted incorrectly

Also
- ivp.py's class docstring was placed incorrectly
- I deleted some whitespaces and cleaned up some code/comments
- there were a couple of errors(?) related to matrix conformability
- fixed a compilation warning message from approximation.py

Best way to review is probably to `make html` in quantecon/docs

### CODE REVIEW

**Comment by mmcky** [comment]
I have added this branch to ``rtd`` for review -- it is currently building.

https://readthedocs.org/projects/quanteconpy/builds/5642386/

File: quantecon/kalman.py [hunk 1]
```diff
@@ -210,15 +234,15 @@ def update(self, y):
 
     def stationary_values(self):
         """
-        Computes the limit of Sigma_t as t  goes to infinity by
-        solving the associated Riccati equation.  Computation is via the
+        Computes the limit of :math:`\Sigma_t` as t goes to infinity by
```
Comment (mmcky) [inline quantecon/kalman.py]: should ``t`` be inline math?

File: quantecon/lqcontrol.py [hunk 2]
```diff
@@ -180,25 +198,27 @@ def update_values(self):
...
         in the infinite horizon case.  Also computes the control matrix
-        F from u = - Fx
+        :math:`F` from :math:`u = - Fx`
 
         Returns
         -------
         P : array_like(float)
             P is part of the value function representation of
-            V(x) = xPx + d
+            :math:`V(x) = x'Px + d`
```
Comment (mmcky) [inline quantecon/lqcontrol.py]: good catch with ``x'``

File: quantecon/lss.py [hunk 3]
```diff
@@ -11,20 +11,25 @@
...
-        
-    The purpose of separating this functionality out is to target it for 
+
+    .. math::
+
+        x_{t+1} = A x_t + v_t
+
+    given :math:`x_0` = x0
+
+    Here :math:`x_t` and :math:`v_t` are both n x 1 and A is n x n.
```
Comment (mmcky) [inline quantecon/lss.py]: should ``A`` be inline math?

File: quantecon/markov/approximation.py [hunk 4]
```diff
@@ -17,46 +17,71 @@
...
 def rouwenhorst(n, ybar, sigma, rho):
-    """
-    Takes as inputs n, p, q, psi.  It will then construct a markov chain
+    r"""
+    Takes as inputs n, p, q, psi. It will then construct a markov chain
     that estimates an AR(1) process of:
-    y_t = \bar{y} + \rho y_{t-1} + \varepsilon_t
-    where \varepsilon_t is i.i.d. normal of mean 0, std dev of sigma
+    :math:`y_t = \bar{y} + \rho y_{t-1} + \varepsilon_t`
+    where :math:`\varepsilon_t` is i.i.d. normal of mean 0, std dev of sigma
```
Comment (mmcky) [inline quantecon/markov/approximation.py]: what is the difference between ``\varepsilon`` and ``\epsilon``?

**Comment by natashawatkins** [comment]
Thanks!

File: quantecon/markov/approximation.py [hunk 5]
```diff
@@ -114,14 +140,16 @@ def row_build_mat(n, p, q):
...
     the linear Gaussian AR(1) process
 
-        y_{t+1} = rho * y_t + u_{t+1}
+    .. math::
+
+        y_{t+1} = \rho y_t + u_{t+1}
 
-    using Tauchen's method.  Here {u_t} is an iid Gaussian process with zero
-    mean.
+    using Tauchen's method. Here :math:`{u_t}`` is an iid Gaussian process
```
Comment (mmcky) [inline quantecon/markov/approximation.py]: should this have two end ticks?

File: quantecon/matrix_eqn.py [hunk 6]
```diff
@@ -96,15 +98,18 @@ def solve_discrete_lyapunov(A, B, max_it=50, method="doubling"):
...
 
         X = A'XA - (N + B'XA)'(B'XB + R)^{-1}(N + B'XA) + Q
 
     via a modified structured doubling algorithm.  An explanation of the
     algorithm can be found in the reference below.
 
     Note that SciPy also has a discrete riccati equation solver.  However it
-    cannot handle the case where R is not invertible, or when N is nonzero.
-    Both of these cases can be handled in the algorithm implemented below.
+    cannot handle the case where :math:`R` is not invertible, or when N is
```
Comment (mmcky) [inline quantecon/matrix_eqn.py]: should ``N`` be inline math?

**Comment by jstac** [comment]
@natashawatkins Thanks!  Great work.  I have checked all details but it's a big step forward even if some errors remain.

@mmcky Is the maintainer of the package so I'll leave the merge up to him.

File: quantecon/markov/approximation.py (hunk 5 above)
Comment (natashawatkins) [inline quantecon/markov/approximation.py]: Fixed thanks

File: quantecon/matrix_eqn.py (hunk 6 above)
Comment (natashawatkins) [inline quantecon/matrix_eqn.py]: Fixed thanks

File: quantecon/markov/approximation.py (hunk 4 above)
Comment (natashawatkins) [inline quantecon/markov/approximation.py]: Apparently they look slightly different but represent the same symbol - https://tex.stackexchange.com/questions/304534/what-does-the-var-prefix-stand-for-in-varphi-and-varepsilon-etc

File: quantecon/lss.py (hunk 3 above)
Comment (natashawatkins) [inline quantecon/lss.py]: Fixed thanks

File: quantecon/kalman.py (hunk 1 above)
Comment (natashawatkins) [inline quantecon/kalman.py]: I'm not sure about this - it could be? But maybe unnecessary?

**Comment by mmcky** [comment]
Thanks @natashawatkins.

File: quantecon/lqcontrol.py [hunk 7]
```diff
@@ -19,36 +19,50 @@ class LQ(object):
...
          x_{t+1} = A x_t + B u_t + C w_{t+1}
 
-    Here x is n x 1, u is k x 1, w is j x 1 and the matrices are
-    conformable for these dimensions.  The sequence {w_t} is assumed to
-    be white noise, with zero mean and E w_t w_t = I, the j x j
-    identity.
+    Here :math:`x` is n x 1, :math:`u` is k x 1, :math:`w` is j x 1 and the 
+    matrices are conformable for these dimensions.  The sequence :math:`{w_t}`
+    is assumed to be white noise, with zero mean and
+    :math:`\mathbb{E} [ w_t' w_t ] = I`, the j x j identity.
```
Comment (natashawatkins) [inline quantecon/lqcontrol.py]: Sorry I forgot to click submit review

**Pull request merged**